CHANGELOG.unreleased.md

```diff
@@ -12,6 +12,7 @@
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
 - [Self,IP pools] Fixed the creation being stuck and freezing XO (PR [#4776](https://github.com/vatesfr/xen-orchestra/pull/4776))
+- [New VM] Boot firmware: show confirmation modal for CH/XCP-ng 8.0 hosts [#4749](https://github.com/vatesfr/xen-orchestra/issues/4749) (PR [#4780](https://github.com/vatesfr/xen-orchestra/pull/4780))
```

Comment:
Is a Changelog entry necessary?
Also, we already did show a confirmation modal for this, we're only removing it for 8.1+.

Reply:
I think it's necessary for the user to know that the modal will only be displayed for CH / XCP-ng 8.0 (fix)